dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -164,6 +164,10 @@
     E3018( "Invalid user access string: `{0}`" ),
     E3019( "Sharing is not enabled for this object `{0}`" ),
     E3020( "You must have permissions to create user, or ability to manage at least one user group for the user." ),
+    E3021( "Cannot disable 2FA for yourself with this API endpoint." ),
```

Comment:
The wording "with this API endpoint" is obsolete I think and can be removed from the error message.